helm/vitess/templates/_orchestrator.tpl

```diff
@@ -24,34 +24,6 @@ spec:
 
 ---
 
-###################################
```

Comment:
We should also remove the `serviceName` from the StatefulSet.

Reply:
Actually it looks like we have to set it to empty since it's required.

Reply:
I actually changed it to the main `orchestrator` service, since that is replacing the old headless service. I also wonder if that will solve your port problem. I'd test, but AKS is still not working.